notebooks/00_quick_start/fastai_recommendation.ipynb

```diff
@@ -0,0 +1,788 @@
+   "cell_type": "code",
+   "execution_count": 35,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# fix random seeds to make sure out runs are reproducible\n",
+    "np.random.seed(101)\n",
+    "torch.manual_seed(101)\n",
+    "torch.cuda.manual_seed_all(101)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 36,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "start_time = time.time()\n",
+    "\n",
+    "data = CollabDataBunch.from_df(ratings_df, pct_val=0.25, user_name=USER, item_name=ITEM, rating_name=RATING)\n",
+    "\n",
+    "preprocess_time = time.time() - start_time"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 37,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<table>  <col width='10%'>  <col width='10%'>  <col width='10%'>  <tr>\n",
+       "    <th>UserId</th>\n",
+       "    <th>MovieId</th>\n",
+       "    <th>target</th>\n",
+       "  </tr>\n",
+       "  <tr>\n",
+       "    <th>785</th>\n",
+       "    <th>423</th>\n",
+       "    <th>2.0</th>\n",
+       "  </tr>\n",
+       "  <tr>\n",
+       "    <th>110</th>\n",
+       "    <th>230</th>\n",
+       "    <th>3.0</th>\n",
+       "  </tr>\n",
+       "  <tr>\n",
+       "    <th>72</th>\n",
+       "    <th>553</th>\n",
+       "    <th>5.0</th>\n",
+       "  </tr>\n",
+       "  <tr>\n",
+       "    <th>871</th>\n",
+       "    <th>202</th>\n",
+       "    <th>4.0</th>\n",
+       "  </tr>\n",
+       "  <tr>\n",
+       "    <th>268</th>\n",
+       "    <th>558</th>\n",
+       "    <th>3.0</th>\n",
+       "  </tr>\n",
+       "</table>\n"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "data.show_batch()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we will create a `collab_learner` for the data. We will be using 40 latent factors. This will create an embedding for the users and the items that will map each of these to 40 floats as can be seen below. Note that the embedding parameters are not predefined, but are learned by the model.\n",
+    "\n",
+    "In addition, we are setting the range of possible ratings to a range from 0 to 5.5 and set a value for weight-decay for regularization."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 38,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "EmbeddingDotBias(\n",
+       "  (u_weight): Embedding(944, 40)\n",
+       "  (i_weight): Embedding(1628, 40)\n",
+       "  (u_bias): Embedding(944, 1)\n",
+       "  (i_bias): Embedding(1628, 1)\n",
+       ")"
+      ]
+     },
+     "execution_count": 38,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "learn = collab_learner(data, n_factors=40, y_range=[0,5.5], wd=1e-1)\n",
```

Comment:
I think the range is 1-5 for movielens 100k

Reply:
Although ratings can only range from 1-5, I am are setting the range of possible ratings from 0 to 5.5 -- that will allow the model to predict values to both sides of 1 and 5, which improves accuracy.
I added that as an explanation to the cell above